packages/scene-composer/src/components/panels/scene-settings/MatterportTagSync.tsx

```diff
@@ -102,7 +102,7 @@ export const MatterportTagSync: React.FC = () => {
         }
 
         if (oldTagMap[key]) {
-          await handleUpdateMatterportTag(layerId, oldTagMap[key].nodeRef, oldTagMap[key].node, value);
+          await handleUpdateMatterportTag(layerId, rootId, oldTagMap[key].nodeRef, oldTagMap[key].node, value);
```

Comment:
Both for line#85 and #105 looks identitical. Why do we have identitical conditions here?

Reply:
that's existing code that I didn't change logic other than adding one more parameter